docs/adr/0003-change-api-of-add-to-a-list-form.md

```diff
@@ -0,0 +1,86 @@
+# Change API of add-to-a-list form
+
+* Status: accepted
+* Deciders: platui, design resources
+* Raised: 2021-06-22
+* Decided: 2021-07-01
+* Components: add-to-a-list
+
+Technical Story: PLATUI-1212
+
+## Context and Problem Statement
+
+The radio buttons in add-to-a-list are using the translated strings `messages.yes` and `messages.no` for their value.
+This means when the form is submitted the handler services must understand and respond correctly to both "Yes/No" and 
+"Iawn/Na". For teams this may be easy to forget and miss from test coverage. Maintainers of the implementation may also 
+not realise that changing the translation could break handling of the submitted form for services using this pattern.
+
+Code example from nunjucks implementation:
+```
+  ...
+  {{ govukRadios({
+    classes: "govuk-radios--inline",
+    idPrefix: "add-another",
+    name: "add-another",
+    fieldset: {
+      legend: {
+        text:  messages.addAnother.prefix + " " + params.itemType.singular | default(messages.itemTypeDefault.singular) + messages.addAnother.suffix + "?",
+        isPageHeading: false,
+        classes: "govuk-fieldset__legend govuk-fieldset__legend--m"
+      }
+    },
+    hint: {
+      text: params.hintText
+    },
+    items: [
+      { text: messages.yes, value: messages.yes },
+      { text: messages.no, value: messages.no }
+    ] }) }}
+    ...
+```
+
+Taken from: https://github.com/hmrc/hmrc-frontend/blob/ed7d1b99009f3e032ec6553bdc6d518b4d1a66f8/src/components/add-to-a-list/template.njk#L82-L83
+
+## Decision Drivers
+
+* we want to have a stable predictable interface for implementers to work to
+* reduce chance that the consequences of changes can be missed
+
+## Considered Options
+
+* Option 1: do nothing
+* Option 2: change to "true" and "false"
+* Option 3: change to "Yes" and "No" (capitalized)
+
+## Decision Outcome
```

Comment:
Do you think we should come down on a decision before publishing the ADR? Or make a note to amend this once it's decided? Thinking of future documentation value.

Reply:
I think it's probably a good idea to publish the decision now but I could change the status to "accepted, final implementation not yet decided" and have it as a criteria of the ticket to implement the change to update this. What do you think @JoPintoPaul?